lib/editor.js

```diff
@@ -8,6 +8,8 @@ var lineNum = 0;
 var lastLine;
 var showStatus = global.appconfig.status;
 
+var noColor = (process.argv.indexOf('--no-color') !== -1);
```

Comment:
I use minimist parse cmd args in index.js and put them in the global.appconfig object. For example like the show status flag..